src/RenderWebGL.js

```diff
@@ -623,15 +623,19 @@ class RenderWebGL extends EventEmitter {
         const drawable = this._allDrawables[drawableID];
         if (!drawable) return null;
 
+        // Convert client coordinates into absolute scratch units
+        const scratchX = this._nativeSize[0] * ((x / this._gl.canvas.clientWidth) - 0.5);
+        const scratchY = this._nativeSize[1] * ((y / this._gl.canvas.clientHeight) - 0.5);
+
         const gl = this._gl;
         twgl.bindFramebufferInfo(gl, this._queryBufferInfo);
 
         const bounds = drawable.getFastBounds();
         bounds.snapToInt();
 
-        // Translate input x and y to coordinates relative to the drawable
-        const pickX = x - ((this._nativeSize[0] / 2) + bounds.left);
-        const pickY = y - ((this._nativeSize[1] / 2) - bounds.top);
+        // Translate to scratch units relative to the drawable
+        const pickX = scratchX - bounds.left;
+        const pickY = scratchY + bounds.top;
```

Comment:
Here, it looks like the coordinate system for `pickX` and `pickY` is changing? Does this require changes elsewhere?